clients/venice-thin-client/src/main/java/com/linkedin/venice/client/stats/ClientMetricEntity.java

```diff
@@ -36,6 +37,40 @@ public enum ClientMetricEntity implements ModuleMetricEntityInterface {
   RETRY_RESPONSE_KEY_COUNT(
       "retry.response.key_count", MetricType.MIN_MAX_COUNT_SUM_AGGREGATIONS, MetricUnit.NUMBER,
       "Key count of retry responses for client", setOf(VENICE_STORE_NAME, VENICE_REQUEST_METHOD)
+  ),
+
+  /**
+   * Client duplicate key count.
+   */
+  REQUEST_DUPLICATE_KEY_COUNT(
+      "request.duplicate_key_count", MetricType.COUNTER, MetricUnit.NUMBER,
+      "Duplicate key count of requests for client",
+      setOf(VENICE_STORE_NAME, VENICE_REQUEST_METHOD, VENICE_RESPONSE_STATUS_CODE_CATEGORY)
+  ),
+
+  /**
+   * Client request count.
+   */
+  REQUEST_TIMEOUT_REQUESTED_DURATION(
+      "request.timeout.requested_duration", MetricType.MIN_MAX_COUNT_SUM_AGGREGATIONS, MetricUnit.MILLISECOND,
+      "The timeout duration (in milliseconds) that was configured for client Future",
+      setOf(VENICE_STORE_NAME, VENICE_REQUEST_METHOD)
+  ),
+
+  /**
+   * Ratio of timed out keys to total keys in the request.
```

Comment:
This measure the non timedout keys ie the ratio of keys collected before it timed out.

Reply:
Updated.